packages/generate-coa-pdf-template/utils/create-example.ts

```diff
@@ -12,10 +17,10 @@ import styles from './styles';
 
 const fonts = {
   Lato: {
-    normal: `${__dirname}/../../../node_modules/lato-font/fonts/lato-normal/lato-normal.woff`,
-    bold: `${__dirname}/../../../node_modules/lato-font/fonts/lato-bold/lato-bold.woff`,
-    italics: `${__dirname}/../../../node_modules/lato-font/fonts/lato-light-italic/lato-light-italic.woff`,
-    light: `${__dirname}/../../../node_modules/lato-font/fonts/lato-light/lato-light.woff`,
+    normal: `${__dirname}/../../../node_modules/@fontsource/lato/files/lato-latin-400-normal.woff`,
```

Comment:
Why the change from `lato-font` to `fontsource` just out of interest?

Reply:
https://github.com/s1seven/schema-tools/issues/247#issuecomment-1961890353